web_src/js/features/repo-home.js

```diff
@@ -102,9 +105,9 @@ export function initRepoTopicBar() {
         const query = stripTags(this.urlData.query.trim());
         let found_query = false;
         const current_topics = [];
-        $topicDropdown.find('a.label.visible').each((_, el) => {
+        for (const el of topicDropdown.querySelectorAll(':scope > a.label.visible')) {
```

Comment:
```suggestion
        for (const el of topicDropdown.querySelectorAll('a.label.visible')) {
```

`find` doesn't need scope.